docs/api/13_wallets/create-wallet.mdx

```diff
@@ -0,0 +1,245 @@
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+  ```python
+  from lago_python_client import Client
+  from lago_python_client.models import Wallet
+
+  client = Client(api_key='__YOUR_API_KEY__')
+
+  wallet = Wallet(
+    name='wallet_name',
+    rate_amount='1.5',
+    paid_credits='20.0',
+    granted_credits='10.0',
+    expiration_date='2022-07-07',
+    external_customer_id='12345'
+  )
+  client.wallets().create(wallet)
+  ```
+  </TabItem>
+
+  <TabItem value="ruby" label="Ruby">
+
+  ```ruby
+  require 'lago-ruby-client'
+
+  client = Lago::Api::Client.new({api_key: '__YOUR_API_KEY__'})
+
+  client.wallets.create({
+      name: 'wallet_name',
+      rate_amount: '1.5',
+      paid_credits: '20.0',
+      granted_credits: '10.0',
+      expiration_date: '2022-07-07',
+      external_customer_id: '12345'
+  })
+  ```
+  </TabItem>
+
+  <TabItem value="javascript" label="Node.js">
+
+  ```javascript
+  import Wallet from 'lago-nodejs-client/wallet'
+
+  let client = new Client('__API_KEY__')
+
+  let wallet = new Wallet({name: 'wallet_name', rate_amount: '1.5', paid_credits: '20.0',
+    granted_credits: '10.0', expiration_date: '2022-07-07', external_customer_id: '12345'
+  })
+  await client.createWallet(wallet);
+  ```
+  </TabItem>
+
+  <TabItem value="go" label="Go">
+
+  ```go
+    import "fmt"
+    import "github.com/getlago/lago-go-client"
+
+    func main() {
+      lagoClient := lago.New().
+        SetApiKey("__YOUR_API_KEY__")
+
+      walletInput := &lago.WalletInput{
+        Name:               "Wallet Name",
+        RateAmount:         "1.5",
+        PaidCredits:        "20.0"
+        GrantedCredits:     "10.0",
+        ExpirationDate:     "2022-07-07",
+        ExternalCustomerId: "12345",
+      }
+
+      wallet, err := lagoClient.Wallet().Create(walletInput)
+      if err != nil {
+        // Error is *lago.Error
+        panic(err)
+      }
+        
+      // wallet is *lago.Wallet
+      fmt.Println(wallet)
+    }
+  ```
+
+  </TabItem>
+</Tabs>
+
+## Arguments
+
+```json
+{
+  "wallet": {
+    "name": "wallet_name",
+    "rate_amount": "1.5",
+    "paid_credits": "20.0",
+    "granted_credits": "10.0",
+    "expiration_date": "2022-07-07",
+    "external_customer_id": "12345"
+  }
+}
+```
+
+| Attributes | Type | Description |
+|--|--|--|
+| name | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Wallet name |
+| rate_amount | String &nbsp &nbsp &nbsp<Required>**Required**</Required> | Rate between credits and the amount in given currency |
+| paid_credits | String &nbsp &nbsp &nbsp<Optional>**Optional (This field is required only if there is no granted credits)**</Optional> | Paid credits. |
+| granted_credits | String &nbsp &nbsp &nbsp<Optional>**Optional (This field is required only if there is no paid credits)**</Optional> | Granted (free) credits. |
+| expiration_date | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Date that determines when the wallet is expiring. |
```

Comment:
"Date that determines when the wallet will expire."

Reply:
fixed